src/session.ts

```diff
@@ -188,6 +199,135 @@ export function valueSession(): void {
   Session.current().toFile("test.json");
 }
 
+let marginalSession: Session | null = null;
+let marginalSessionDiff: Session | null = null;
+let barfSession: Session | null = null;
+let barfSessionStartTurns = totalTurnsPlayed();
+let continueMeatTracking = true;
+let numTrackedItemTurns: number | null = null;
+export function trackBarfSessionStatistics(): void {
+  // If we are overdrunk, don't track statistics
+  if (myInebriety() > inebrietyLimit()) return;
+
+  // Start barfSession if we have not done so
+  if (!barfSession) {
+    barfSession = Session.current();
+    barfSessionStartTurns = totalTurnsPlayed();
+  }
+
+  // Start tracking items if at least one of these is true
+  // 1) We have run at least 100 barf turns
+  // 2) We have less than 200 adv to run
+  if (
+    !marginalSession &&
+    (totalTurnsPlayed() - barfSessionStartTurns >= 100 ||
+      (myAdventures() <= 200 + 25 + globalOptions.saveTurns &&
+        myAdventures() > globalOptions.saveTurns + 25))
+  ) {
+    marginalSession = Session.current();
+    numTrackedItemTurns = myAdventures() - globalOptions.saveTurns - 25;
+  }
+
+  // Start tracking meat if we have less than 75 turns left
+  // Also create a backup tracker for items
+
+  if (
+    (!get("_garboMarginalMeatCheckpoint") || !get("_garboMarginalMeatTurns")) &&
```

Comment:
similarly, this meat tracker needs some sort of "if the user is only running 75 turns, do not report marginals" as discussed above with the items to ensure it does not encase initial embezzlers & yachtzee. to be clear, it doesn't need to be at these if statements -- it's fine if it tabulates this all regardless, but there should be -something- that shuts down reporting in cases where the reporting will clearly be wrong

Reply:
So in `printMarginalSession()` we only report outliers when we have run sufficiently many turns. We still provide an estimate if we ran less than sufficient turns, but we clearly cannot estimate outliers without sufficient data. I could put in a warning when we do this alternative printout.